src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/abstractjavadoc/InputAbstractJavadocJavadocTagsWithoutArgs.java

```diff
@@ -59,14 +59,14 @@ private void readObject(java.io.ObjectInputStream inputStream) {
     }
 
     /**
-     * {@link} // violation
+     * {@link} // ok
```

Comment:
Unfortunately this doesn't show practical reason to user. As it is not real Check, change from violation to ok , does not mean much.

https://checkstyle.org/config_javadoc.html#SingleLineJavadoc
https://checkstyle.org/config_javadoc.html#SummaryJavadoc

Please extend Inputs for such Checks to show what type of possibilities your fix is opening.

Reply:
Hey, I know this was a big delay from my side.
but sorry I didn't get how this doesn't mean much? there were several parsing errors, and as I see they were solved.
**I know I need to push a single commit and I'm doing this now** I'm just talking about the updated grammar